src/runtime/smart-button-api.js

```diff
@@ -107,6 +108,11 @@ export class SmartSubscriptionButtonApi {
       'width': '100%',
     });
     this.button_.appendChild(this.iframe_);
+    if (this.deps_.config().analyticsMode == AnalyticsMode.IMPRESSIONS) {
+      // isFromUserActsion = False
```

Comment:
At this point you could just start logging isFromUserAction (note mispelling in your comment)

Reply:
Thanks! I'll remove the comment because I'll actually be setting this server-side.